src/chainparams.cpp

```diff
@@ -382,11 +378,25 @@ class SigNetParams : public CChainParams {
     }
 };
 
+void ReadSigNetArgs(const ArgsManager& args, CChainParams::SigNetOptions& options) {
```

Comment:
Seems more ergonomic to write:

```c++
CChainParams::SigNetOptions ReadSigNetArgs(const ArgsManager& args)
{
    ...
```

but maybe this is just deliberately keeping with the same pattern as per https://github.com/bitcoin/bitcoin/pull/26883/files#r1068247937

Reply:
Yes, also keeping it the same as in: https://github.com/bitcoin/bitcoin/pull/25862/files#diff-4e268aeb074a176689aace31957a889c1f39c909e6678ca7be8707bda1a7be56R11